src/Components/Sidebar/index.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./index.css";
+import { SidebarData } from "./SidebarData";
+
+function Sidebar(){
```

Comment:
@Lokendra-egov  why one more new folder created as Components?
can you move back to components folder.

Reply:
Reviewers @ansh-egov  and @aaradhya-egov  make sure you validate each change request thoroughly.

> 1. folder structure should be followed.
> 2. just see if that change is required to be merged, 
> 3. keep some guidelines in the team, for coding , components usage etc.

cc. @digit-training/repo-developers